src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27229.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue27229 : _IssuesUITest
+	{
+		public Issue27229(TestDevice device) : base(device) { }
+
+		public override string Issue => "CollectionView, EmptyView Fills Available Space By Default";
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void CollectionViewEmptyViewFillsAvailableSpaceByDefault()
+		{
+			App.WaitForElement("ReadyToTest");
+			VerifyScreenshot();
```

Comment:
Pending snapshots. Already available in the latest [build](https://dev.azure.com/xamarin/public/_build/results?buildId=133261&view=ms.vss-test-web.build-test-results-tab&runId=3638306&resultId=100096&paneView=debug):
![image](https://github.com/user-attachments/assets/79dfb423-5e9f-4f61-b5d4-39da9503bdab)